relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -200,15 +200,18 @@ pub(crate) fn extract_tags(span: &Span, config: &Config) -> BTreeMap<SpanTagKey,
             .and_then(|data| data.get("description.scrubbed"))
             .and_then(|value| value.as_str());
 
-        // TODO(iker): we're relying on the existance of `http.method`
+        // TODO(iker): we're relying on the existance of `http.method`/`http.response.method`
```

Comment:
Can we remove this TODO now and just put in the doc string the places we actually look in (and maybe why? E.g. open telemetry scheme etc.), which in my opinion makes more sense.

Reply:
Yes I can amend this, good call.